frontend/src/utils/video.ts

```diff
@@ -4,7 +4,7 @@ import { RelatedEntityObject, VideoObject } from './types';
 
 export function extractVideoId(idOrUrl: string) {
   const matchUrl = idOrUrl.match(
-    /(?:https?:\/\/)?(?:www\.)?(?:youtube\.com\/watch\?v=|youtu.be\/)([A-Za-z0-9-_]+)/
+    /(?:https?:\/\/)?(?:www\.)?(?:youtube\.com\/watch\?v=|youtu.be\/|tournesol\.app\/entities\/yt:)([A-Za-z0-9-_]+)/
```

Comment:
I found an old bug in the original regex.

Look the middle path:
 
![capture](https://user-images.githubusercontent.com/39056254/177587285-e7ec79fb-5ec0-4672-bfec-e97e76aee7be.png)

It shouldn't accept any character, only the dot:

![capture2](https://user-images.githubusercontent.com/39056254/177587394-785949a3-2a0c-4453-96a2-7b647cced1e4.png)

Accepting this suggestion will fix the problem.

```suggestion
    /(?:https?:\/\/)?(?:www\.)?(?:youtube\.com\/watch\?v=|youtu\.be\/|tournesol\.app\/entities\/yt:)([A-Za-z0-9-_]+)/
```

Reply:
Fixed by 7b11bff05fb208e35503b6f9c8b8b90425ec2926.